pyproject.toml

```
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2025 Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+[project]
+name = "power-grid-model-workshop"
+version = "0.1.0"
+description = "A collection of Jupyter notebooks for Power Grid Model workshops"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "jupyter>=1.1.1",
+    "matplotlib>=3.10.1",
+    "pandas>=2.2.3",
+    "power-grid-model>=1.10.88",
```

the original `requirements.txt` contains the `doc` optional dependency set. We need to verify that it is not mandatory to have it, or re-add it

```suggestion
    "power-grid-model[doc]>=1.10.88",
```